mi/values.yaml

```diff
@@ -82,45 +87,49 @@ wso2:
         # -- Primary keystore alias
         alias: "wso2carbon"
         # -- Primary keystore password
-        password: ""
+        password: "wso2carbon"
         # -- Primary keystore key password
-        keyPassword: ""
+        keyPassword: "wso2carbon"
       internal:
         # -- Internal keystore file name
         fileName: "wso2carbon.jks"
         # -- Internal keystore alias
         alias: "wso2carbon"
         # -- Internal keystore password
-        password: ""
+        password: "wso2carbon"
         # -- Internal keystore key password
-        keyPassword: ""
+        keyPassword: "wso2carbon"
     trustStore:
       primary:
         # -- Primary truststore file name
         fileName: "client-truststore.jks"
         # -- Primary truststore password
-        password: ""
+        password: "wso2carbon"
   deployment:
     securityContext:
       # -- Enable/Disable AppArmor (https://kubernetes.io/docs/tutorials/security/apparmor/)
-      apparmor: true
+      apparmor: false
```

Comment:
Let's not change these values in the default values.yaml. Better to add these values in the readme as examples.